### PR TITLE
Add security advisories for Symfony CSRF issue

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

**Review by Nyholm — APPROVED** [review]

**Review by jderusse — APPROVED** [review]

File: symfony/framework-bundle/CVE-2022-xxx.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+title:     "CVE-2022-xxx: CSRF token missing in forms"
+link:      https://symfony.com/cve-2022-xxx
```
Comment (nicolas-grekas) [inline symfony/framework-bundle/CVE-2022-xxx.yaml:2]: https://symfony.com/cve-2022-xxxx

File: symfony/framework-bundle/CVE-2022-xxx.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+title:     "CVE-2022-xxxx: CSRF token missing in forms"
+link:      https://symfony.com/cve-2022-xxxx
+cve:       CVE-2022-xxxx
+branches:
+    5.3.x:
+        time:     2022-01-29 12:00:00
+        versions: ['5.3.14', '5.3.14']
```
Comment (jderusse) [inline symfony/framework-bundle/CVE-2022-xxx.yaml:7]: maybe with a sign like `=` ?
Or `>=5.3.24, <=5.3.24`

Comment (jderusse) [inline symfony/framework-bundle/CVE-2022-xxx.yaml:7]: maybe with a sign like `=` ?
Or `>=5.3.24, <=5.3.24`

Comment (nicolas-grekas) [reply symfony/framework-bundle/CVE-2022-xxx.yaml:7]: why? this would be exactly the same isn't it?

Comment (jderusse) [reply symfony/framework-bundle/CVE-2022-xxx.yaml:7]: see CI check

**Review by jderusse — APPROVED** [review]

**Review by michaelcullum — APPROVED** [review]

**Pull request merged**